http/h2_stream.lua

```diff
@@ -1386,7 +1386,7 @@ function stream_methods:write_chunk(payload, end_stream, timeout)
 		if max_available < (#payload - sent) then
 			if max_available > 0 then
 				-- send partial payload
-				local ok, err, errno = self:write_data_frame(payload:sub(sent+1, sent+max_available), false, timeout)
+				local ok, err, errno = self:write_data_frame(payload:sub(sent+1, sent+max_available), end_stream, false, timeout)
```

Comment:
Shouldn't `end_stream` be always `false` here?
The final frame below should be the only one to consider the parameter.

Reply:
Not by `:write_chunk` which is a high-level interface.